plone/app/portlets/portlets/actions.pt

```diff
@@ -4,15 +4,15 @@
       tal:omit-tag=""
       i18n:domain="plone">
 
-<section class="portlet portletActions" role="menu"
+<div class="portlet portletActions"
```

Comment:
aside is not expected to be related with the main content. from the specs:

> The aside element represents a section of a page that consists of content that is tangentially related to the content around the aside element, and which could be considered separate from that content. Such sections are often represented as sidebars in printed typography.

Also, our portlets are already wrapped in an aside column. But I consider that as not optimal, as the content is only for layout purposes.
Each aside introduces a new section. The examples in the w3 document show grouped "portlets" in an aside tag, but also multiple aside tags for one side bar. Since we don't have a defined semantic relationship between portlets and hardly can establish one, wrapping each portlet into an aside tag sounds good to me.

HTML 5 leaves much room for interpretation. Are there other opinions? /cc @polyester @davilima6 anyone else?

But if this blocks merging your pull-request, I'm totally OK with your approach. I try to prepare a PLIP for simplifying and semantifying our HTML layout, which will cover also the portlets topic.